src/content/self-hosted/administration/configuration.mdx

```diff
@@ -150,6 +150,10 @@ The `allowDomains` field is optional. Only users with Google Workspace accounts
 
 Follow your OpenID Connect service provider's documentation on how to create the required application.
 
+We have custom guides for:
```

Comment:
Okteto supports any identity provider that implements the OpenID Connect standard. The following external identity providers have been tested with Okteto: